Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+COPY . /app
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE 50051
```

Comment:
I think se shouldn't have EXPOSE rule in the dockerfile. The docker compose is a better place